src/scenic/domains/driving/model.scenic

```diff
@@ -259,8 +259,9 @@ class Vehicle(DrivingObject):
             :obj:`Color.defaultCarColor`.
     """
     regionContainedIn: roadOrShoulder
-    position: Point on road
-    heading: (roadDirection at self.position) + self.roadDeviation
+    position: new Point on road
+    heading: roadDirection at self.position
+    yaw: self.roadDeviation
```

Comment:
This doesn't look right. If we're sticking with 2D mode for the driving domain for now then we should just keep the old default value for `heading` and not specify `yaw` at all. (I know this should work since `heading` gets mapped to `parentOrientation` in 2D mode, but the code looks contradictory and we shouldn't encourage others to write code like it. Either you should write for Scenic 2, specifying only `heading`, or for 3, specifying `yaw` etc. but not `heading`.)

Reply:
Ah good catch. I think Matthew made these changes before we had 2D mode, but now that we do I'll swap them back.